### PR TITLE
website/integrations: apache guacamole: cleanup doc page

### PR DESCRIPTION
**Pull request by dominic-r**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

**Review by tanberry — APPROVED** [review]
Thank you!! Yep, "loose" versus "lose" is a doozy; I'm glad you caught that. And capitalizing Docker. Appreciate your help.

**Comment by tanberry** [comment]
Hmmmm the `lint (prettier-check)` build check is failing. Can you try running `make website` again and pushing to PR? If that doesn't fix it I'll ask on of the engineers to look at it. (We can safely ignore the other fails.)

**Comment by tanberry** [comment]
I think it might be something on our side... let me talk to eng about it tomorrow.

File: website/integrations/services/apache-guacamole/index.mdx [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Apache Guacamole™
 
 ## What is Apache Guacamole™
 
-> Apache Guacamole is a clientless remote desktop gateway. It supports standard protocols like VNC, RDP, and SSH.
+> Apache Guacamole is a clientless remote desktop gateway that supports standard protocols like VNC, RDP, and SSH.
```
Comment (BeryJu) [inline website/integrations/services/apache-guacamole/index.mdx:9]: This is a verbatim quote from the website and as such it should stay
```suggestion
> Apache Guacamole is a clientless remote desktop gateway. It supports standard protocols like VNC, RDP, and SSH.
```

**Pull request merged**